src/manifest.json.tpl

```diff
@@ -25,7 +25,7 @@
 
   "browser_action": {
     "default_icon": {
-      "32": "icons/icon-lockbox.svg"
+      "32": "icons/icon-lockwise.svg"
```

Comment:
this is the line that changes to use the generic key icon instead of the app icon ...